### DISCUSSION TITLE
Welcome to imdone Discussions!

### DISCUSSION THREAD

**Discussion by piascikj** [imdone/imdone#37] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alainmuls** [comment]
Hi, I just started with the free trial. 

I used a markdown file containing:
- __Action Items__
	+ [ ] NMEA GNSS message selection ... AM / OD / PJDM
		* [ ] NMEA version selection
		* [ ] NMEA messages for PNT
		* [ ] NMEA messages for interference detection
	+ [ ] Determination of KPIs ... AM / OD / PJDM
		* [ ] related to PNT accuracy
		* [ ] related to GNSS signals / services availability
		* [ ] related to Continuity of Service (CoS)
		* [ ] related to Integrity
		* [ ] document automotive industry to distribute ... OD

these tasks.

PS: I also tried with removing the first line and shifting the main tasks to the beginning of the line.

By opening the directory with only this file in it, nothing happened.

Then I decided to create a separate directory (imdone) with nothing in there and added manually into 2 tasks the 2 main action items (with its subtasks) using the builtin editor for the tasks. They do show up, but:
- clicking on a [ ]  didn't go to [x] (so subtask completed)
- I want also to assign the task to a person, is there a sort of @alan that we can add?
- I could not color a card since the save button was not activated after a color setting was added

How does I add a due-date? 

Tx/ALain
